packages/@aws-cdk/aws-stepfunctions-tasks/lib/emrcontainers/utils/role-policy/index.py

```diff
@@ -0,0 +1,14 @@
+import subprocess as sp
+import os
+import logging
+
+def handler(event, context):
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+    command = f"/opt/awscli/aws emr-containers update-role-trust-policy --cluster-name {event['ResourceProperties']['eksClusterId']} --namespace {event['ResourceProperties']['eksNamespace']} --role-name {event['ResourceProperties']['roleName']}"
```

Comment:
Can you add the link to: https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-stepfunctions#custom-state in a comment here as well?